content/_guides/creatingchannels.md

```diff
@@ -1,52 +1,211 @@
+branding rules you need to comply with.
+
+Congratulations on your new Libera.Chat channel!
+
+### Troubleshooting
+
+If you do not get the Operator status on join, it could be because there are
+people already in the channel. You will need to ask them to leave while you
+set it up. If they do not cooperate, you will likely need to pick a new name.
+
+If the channel is already registered but not in use, or there are no active
+Operators, it may be subject to [expiry](/policies#channels), depending on the
+type of namespace it belongs to. As part of the expiry process, staff will
+take input from both you and any existing channel members.
+
+If you are a representative of a project and you wish to use a channel that is
+already registered, you can ask staff about a [group registration](chanreg).
+
+If you are unsure of any of the above, talk with Libera.Chat staff. You can
+find staff either in `#libera` or on `/stats p`.
+
+Once you have resolved the conflict, you can resume
+[the registration process](creatingchannels#registering-with-chanserv).
+
+### Special Instructions for Group Contacts
+
+Libera.Chat uses an Atheme feature called `ProjectServ` to track group
+registrations. It shows some additional information in `NickServ` and allows
+for convenient features such as `CLAIM` in `ChanServ`.
+
+If you already have a group registration set up with us, and you are listed as
+a Group Contact (GC) for the namespace you have chosen, it will be displayed
+to you in your `NickServ` info. Look for a line like:
+
+```irc
+NickServ: Group contact for projectname (#projectname; projectname/*)
+```
+
+`CLAIM` will allow you to acquire a channel _within your namespace_ even if
+that channel is already registered to someone else, without the need for staff
+intervention. It can also be used for initial registration even if it is not
+already taken.
+
+```irc
+09:23 -- ChanServ: ***** ChanServ Help *****
+09:23 -- ChanServ:  
+09:23 -- ChanServ: Help for CLAIM:
+09:23 -- ChanServ:  
+09:23 -- ChanServ: The CLAIM command allows you to take control of a channel
+09:23 -- ChanServ: belonging to a project you are authorized to represent.
+09:23 -- ChanServ:  
+09:23 -- ChanServ: Syntax: CLAIM <#channel>                            
+09:23 -- ChanServ:  
+09:23 -- ChanServ: Examples:
+09:23 -- ChanServ:     /msg ChanServ CLAIM #coolproject-dev            
+09:23 -- ChanServ:  
+09:23 -- ChanServ: ***** End of Help *****
+```
+
+Using `CLAIM` to grab a channel from someone else might be seen as hostile, or
+it could confuse people, so it is a good idea to at least give some warning.
+Try to ask for the cooperation of people who have already occupied or
+registered a channel before taking it from them.
+
+## Setting channel modes
+
+Channels differ in what `cmodes` they need. Refer to
+[the channel modes guide](channelmodes) for detailed information about them.
+
+The default modes set on a new channel are `+Cnst`.
```

Comment:
Have you checked that these are the current default modes? They are a good default but I don't think they are the solanum default. We might need to change this here or in the server config.

Reply:
`+Cnst` are indeed the default modes (confirmed just now)

Reply:
Oh yeah, changed a bit over a week ago. Good news.